external/os-guides/pentesting/kali.md

```diff
@@ -6,595 +6,168 @@ redirect_from:
-access can change this configuration in firewall settings for the TemplateVM.
-
-**Note:** The prompt on each line indicates where each command should be entered
-(`@dom0`, `@kali-rolling`, `@xxxx-dvm` or `@debian-<X>`).
-
-1. Retrive the Kali Linux GPG key using a DispVM.
-
-        [user@xxxx-dvm ~]$ gpg --keyserver hkp://keys.gnupg.net --recv-key 44C6513A8E4FB3D30875F758ED444FF07D8D0BF6
-        [user@xxxx-dvm ~]$ gpg --list-keys --with-fingerprint 44C6513A8E4FB3D30875F758ED444FF07D8D0BF6 
-        [user@xxxx-dvm ~]$ gpg --export --armor 44C6513A8E4FB3D30875F758ED444FF07D8D0BF6 > kali-key.asc
-
-2. **DO NOT TURN OFF** the DispVM, the `kali-key.asc` file will be copied to
-   the Kali Linux template in a further step.
-
-3. Make sure the key is the authentic Kali key.
-   See the [Kali website] for further advice and instructions on verification.
-
-#### Create a Kali Linux (rolling) template
-
-These instructions will show you how to upgrade a Debian TemplateVM to Kali Linux.
-
-1. (Optional) Check for latest Debian stable template and install it (if not already done)
-
-        [user@dom0 ~]$ sudo qubes-dom0-update --action="search all" qubes-template-debian
-        [user@dom0 ~]$ sudo qubes-dom0-update <latest Debian template>
-
-2. Start, update and close your latest Debian template
-
-        [user@dom0 ~]$ qvm-start debian-<X>
-        [user@dom0 ~]$ qvm-run -a debian-<X> gnome-terminal
-        [user@debian-<X> ~]$ sudo apt-get update
-        [user@debian-<X> ~]$ sudo apt-get upgrade
-        [user@dom0 ~]$ qvm-shutdown debian-<X>
-
-3. Clone `debian-X` template
-
-        [user@dom0 ~]$ qvm-clone debian-<X> kali-rolling
-
-4. Check the name of currently used repository in `/etc/apt/sources.list` and current testing [Debian release][Debian-releases]. Update repository list accordingly
-
-        [user@kali-rolling ~]$ sudo sed -i 's/<current stable>/<current testing>/g' /etc/apt/sources.list
-        [user@kali-rolling ~]$ sudo sed -i 's/<current stable>/<current testing>/g' /etc/apt/sources.list.d/qubes-r<X>.list
-      
-      e.g. in this example we update `stretch` stable repository to `buster` testing repository
-      
-        [user@kali-rolling ~]$ sudo sed -i 's/stretch/buster/g' /etc/apt/sources.list
-        [user@kali-rolling ~]$ sudo sed -i 's/stretch/buster/g' /etc/apt/sources.list.d/qubes-r<X>.list
+```shell_session
+# sed -i 's/buster/bullseye/g' /etc/apt/sources.list
+# sed -i 's/buster/bullseye/g' /etc/apt/sources.list.d/qubes-r<X>.list
+```
         
 For installation based on Debian 10 stable, please note that the security repository of Debian testing has [recently been renamed][Debian-security-naming-convention] from `<current testing>/update` to `<current-testing>-security`. To account for that change, execute the following command.
 
-        [user@kali ~]$ sudo sed -i 's/bullseye\/updates/bullseye-security/g' /etc/apt/sources.list
+```shell_session
```

Comment:
Escaping is ugly - you can use *any* delimiter in a sed statement, so s%bullseye/updates%bullseye-security% is a little clearer.